src/pps/pps.proto

```diff
@@ -719,15 +719,12 @@ message CreatePipelineRequest {
   bool autoscaling = 30;
   repeated Toleration tolerations = 34;
   ResourceSpec sidecar_resource_requests = 35;
-  string details_json = 36;
+  reserved 36;
```

Comment:
Well, kind of the opposite: it makes it an error for the same field name and/or field number to be used in the future. It’s a way of saying “we used to have a field named ‘details_json’ with number 36; we don’t any longer; don’t reuse it.’ This is important, because protobufs should be forwards- and backwards-compatible. It would be really bad if a future program reused a field number, then consumed an old stored proto message: it would think that it was reading the new field when really it was reading the old one. They might not even be the same type …

Here’s more documentation: https://protobuf.dev/programming-guides/proto3/#fieldreserved